easybuild/easyconfigs/n/NextDenovo/NextDenovo-2.5.2-20240510-GCCcore-12.3.0.eb

```
@@ -0,0 +1,54 @@
+easyblock = 'MakeCp'
+
+name = 'NextDenovo'
+version = '2.5.2-20240510'
+local_commit = '0e5fa5d'
+
+homepage = 'https://github.com/Nextomics/NextDenovo'
+description = 'NextDenovo is a string graph-based de novo assembler for long reads.'
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://github.com/Nextomics/NextDenovo/archive/']
+sources = [{'download_filename': '0e5fa5d.tar.gz', 'filename': SOURCE_TAR_GZ}]
```

use `local_commit` here:
```suggestion
sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
```